src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -4,40 +4,29 @@
 
 public class FigureSupplier {
     private static final int RANDOM_COUNTER = 10;
+    private static final int FIGURES_QUANTITY = 5;
     private Random random = new Random();
+    private String color = ColorSupplier.getRandomColor();
+    private int randomValue = getRandomNumber();
 
-    private int getSmallerRandomNumber() {
+    private int getRandomNumber() {
         return random.nextInt(RANDOM_COUNTER) + 1;
     }
 
-    private int getBiggerRandomNumber() {
-        return random.nextInt(RANDOM_COUNTER) + 11;
-    }
-
     public Figure getRandomFigure() {
-        String[] figure = {"Circle", "IsoscelesTrapezoid", "Rectangle", "RightTriangle", "Square"};
-        int i = random.nextInt(figure.length);
+        int i = random.nextInt(FIGURES_QUANTITY);
         switch (i) {
             case 0 :
-                return new Circle(ColorSupplier.getRandomColor(),
-                        getSmallerRandomNumber());
+                return new Circle(color, randomValue);
```

Comment:
```suggestion
				int radius = random.nextInt(RANDOM_COUNTER);
                return new Circle(color, radius);
```
and create a private method for this. same to all figures